recipes/westpa/post-link.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+{
+    cd "${PREFIX}/${PKG_NAME}-${PKG_VERSION}"
+    WEST_PYTHON=$(which python2.7)
+    WEST_PYTHON=$WEST_PYTHON $WEST_PYTHON .westpa_gen.py
+    chmod +x westpa.sh
```

Comment:
Sorry, just trying to understand what this is doing? Any reason this couldn't be handled in the install?

Reply:
It's creating a little file that exports some environment variables:

https://github.com/westpa/westpa/blob/master/.westpa_gen.py

It's actually created during the `build` in the `setup.sh`, but the paths are incorrect so we need it to run on the user's system after the install. 

I'm not sure what you mean by "handled in the install". Can you clarify?

Reply:
So one way to simplify this would be to use the `CONDA_PREFIX` environment variable. This is set by `conda` when running `activate` to point to the top of the environment that is activated.